### PR TITLE
docs: improve docs

### PR DESCRIPTION
**Pull request by wizardofhogwarts**

### CODE REVIEW

File: src/content/guides/code-splitting.md [hunk 1]
```diff
@@ -329,8 +330,8 @@ T> Using webpackPreload incorrectly can actually hurt performance, so be careful
 Once you start splitting your code, it can be useful to analyze the output to check where modules have ended up. The [official analyze tool](https://github.com/webpack/analyse) is a good place to start. There are some other community-supported options out there as well:
 
 - [webpack-chart](https://alexkuz.github.io/webpack-chart/): Interactive pie chart for webpack stats.
-- [webpack-visualizer](https://chrisbateman.github.io/webpack-visualizer/): Visualize and analyze your bundles to see which modules are taking up space and which might be duplicates.
-- [webpack-bundle-analyzer](https://github.com/webpack-contrib/webpack-bundle-analyzer): A plugin and CLI utility that represents bundle content as convenient interactive zoomable treemap.
+- [webpack-visualizer](https://chrisbateman.github.io/webpack-visualizer/): Visualize and analyze your bundles to see which modules are taking up space and which might be duplicate.
```
Comment (EugeneHlushko) [inline src/content/guides/code-splitting.md]: Are you sure about this one? Modules (many) -> duplicates.

File: src/content/guides/code-splitting.md [hunk 2]
```diff
@@ -48,7 +49,7 @@ There are three general approaches to code splitting available:
 
 ## Entry Points
 
-This is by far the easiest, and most intuitive, way to split code. However, it is more manual and has some pitfalls we will go over. Let's take a look at how we might split another module from the main bundle:
+This is by far the easiest and most intuitive way to split code. However, it is more manual and has some pitfalls we will go over. Let's take a look at how we might split another module from the main bundle:
```
Comment (EugeneHlushko) [inline src/content/guides/code-splitting.md]: I think there has to be a comma, but after and before `by far`. Any grammar experts around?

File: src/content/concepts/hot-module-replacement.md [hunk 3]
```diff
@@ -64,7 +64,7 @@ A `check` makes an HTTP request to the update manifest. If this request fails, t
 
 The `apply` method flags all updated modules as invalid. For each invalid module, there needs to be an update handler in the module or in its parent(s). Otherwise, the invalid flag bubbles up and invalidates parent(s) as well. Each bubble continues until the app's entry point or a module with an update handler is reached (whichever comes first). If it bubbles up from an entry point, the process fails.
 
-Afterwards, all invalid modules are disposed (via the dispose handler) and unloaded. The current hash is then updated and all `accept` handlers are called. The runtime switches back to the `idle` state and everything continues as normal.
+Afterward, all invalid modules are disposed (via the dispose handler) and unloaded. The current hash is then updated and all `accept` handlers are called. The runtime switches back to the `idle` state and everything continue as normal.
```
Comment (EugeneHlushko) [inline src/content/concepts/hot-module-replacement.md]: Afterwards was just fine imo, requesting help from another grammar expert 😓 Also continues sound ok

File: src/content/guides/code-splitting.md (hunk 2 above)
Comment (wizardofhogwarts) [reply src/content/guides/code-splitting.md]: @EugeneHlushko yes, I also think after `by far` it makes sense, but then we must have it after `intuitive` as well.

File: src/content/concepts/hot-module-replacement.md (hunk 3 above)
Comment (wizardofhogwarts) [reply src/content/concepts/hot-module-replacement.md]: @EugeneHlushko According to me, `modules` being a cluster or plural, `afterwards` with it isn't grammatically correct, same for `continues`. It is subject-verb agreement. Though many a time, afterwards is considered British, not American.

File: src/content/guides/code-splitting.md (hunk 1 above)
Comment (wizardofhogwarts) [reply src/content/guides/code-splitting.md]:
```suggestion
- [webpack-visualizer](https://chrisbateman.github.io/webpack-visualizer/): Visualize and analyze your bundles to see which modules are taking up space and which are maybe duplicates.
```
@EugeneHlushko Though, might is used with past. New solution makes more sense.

File: src/content/concepts/hot-module-replacement.md (hunk 3 above)
Comment (rishabh3112) [reply src/content/concepts/hot-module-replacement.md]: @wizardofhogwarts `afterwards` makes more sense in this context IMO.
```suggestion
Afterwards, all invalid modules are disposed (via the dispose handler) and unloaded. The current hash is then updated and all `accept` handlers are called. The runtime switches back to the `idle` state and everything continue as normal.
```
cc @EugeneHlushko

File: src/content/guides/code-splitting.md (hunk 2 above)
Comment (rishabh3112) [reply src/content/guides/code-splitting.md]:
```suggestion
This is by far, the easiest and the most intuitive way to split code. However, it is more manual and has some pitfalls we will go over. Let's take a look at how we might split another module from the main bundle:
```

**Comment by wizardofhogwarts** [comment]
@rishabh3112 @EugeneHlushko @montogeek Please review. Any other changes required?

File: src/content/guides/code-splitting.md [hunk 4]
```diff
@@ -329,8 +330,8 @@ T> Using webpackPreload incorrectly can actually hurt performance, so be careful
 Once you start splitting your code, it can be useful to analyze the output to check where modules have ended up. The [official analyze tool](https://github.com/webpack/analyse) is a good place to start. There are some other community-supported options out there as well:
 
 - [webpack-chart](https://alexkuz.github.io/webpack-chart/): Interactive pie chart for webpack stats.
-- [webpack-visualizer](https://chrisbateman.github.io/webpack-visualizer/): Visualize and analyze your bundles to see which modules are taking up space and which might be duplicates.
-- [webpack-bundle-analyzer](https://github.com/webpack-contrib/webpack-bundle-analyzer): A plugin and CLI utility that represents bundle content as convenient interactive zoomable treemap.
+- [webpack-visualizer](https://chrisbateman.github.io/webpack-visualizer/): Visualize and analyze your bundles to see which modules are taking up space and which are maybe duplicates.
```
Comment (superburrito) [inline src/content/guides/code-splitting.md]: This change is not needed.

File: src/content/concepts/hot-module-replacement.md [hunk 5]
```diff
@@ -64,7 +64,7 @@ A `check` makes an HTTP request to the update manifest. If this request fails, t
 
 The `apply` method flags all updated modules as invalid. For each invalid module, there needs to be an update handler in the module or in its parent(s). Otherwise, the invalid flag bubbles up and invalidates parent(s) as well. Each bubble continues until the app's entry point or a module with an update handler is reached (whichever comes first). If it bubbles up from an entry point, the process fails.
 
-Afterwards, all invalid modules are disposed (via the dispose handler) and unloaded. The current hash is then updated and all `accept` handlers are called. The runtime switches back to the `idle` state and everything continues as normal.
+Afterwards, all invalid modules are disposed (via the dispose handler) and unloaded. The current hash is then updated and all `accept` handlers are called. The runtime switches back to the `idle` state and everything continue as normal.
```
Comment (superburrito) [inline src/content/concepts/hot-module-replacement.md]: This change is not needed. `... everything continues` / `... everything will continue` are grammatically correct options.

File: src/content/guides/code-splitting.md [hunk 6]
```diff
@@ -48,7 +49,7 @@ There are three general approaches to code splitting available:
 
 ## Entry Points
 
-This is by far the easiest, and most intuitive, way to split code. However, it is more manual and has some pitfalls we will go over. Let's take a look at how we might split another module from the main bundle:
+This is by far, the easiest and the most intuitive way to split code. However, it is more manual and has some pitfalls we will go over. Let's take a look at how we might split another module from the main bundle:
```
Comment (superburrito) [inline src/content/guides/code-splitting.md]:
```suggestion
This is by far the easiest and most intuitive way to split code. However, it is more manual and has some pitfalls we will go over. Let's take a look at how we might split another module from the main bundle:
```

Comment (EugeneHlushko) [reply src/content/guides/code-splitting.md]: 👍

**Comment by EugeneHlushko** [comment]
@wizardofhogwarts please see @superburrito comments

File: src/content/concepts/hot-module-replacement.md (hunk 5 above)
Comment (wizardofhogwarts) [reply src/content/concepts/hot-module-replacement.md]: @superburrito ok

File: src/content/guides/code-splitting.md (hunk 4 above)
Comment (wizardofhogwarts) [reply src/content/guides/code-splitting.md]: @EugeneHlushko @montogeek @rishabh3112 What do you think?

Comment (wizardofhogwarts) [reply src/content/guides/code-splitting.md]: @EugeneHlushko @montogeek @rishabh3112 What do you think?

**Comment by wizardofhogwarts** [comment]
> @wizardofhogwarts please see @superburrito comments

Seen and implemented two of the suggested changes.
Check this:
https://github.com/webpack/webpack.js.org/pull/2891#discussion_r271197064

File: src/content/guides/code-splitting.md (hunk 4 above)
Comment (EugeneHlushko) [reply src/content/guides/code-splitting.md]: i like `might be` better, in terms of how it reads

Comment (wizardofhogwarts) [reply src/content/guides/code-splitting.md]: @EugeneHlushko suggestion implemented

File: src/content/guides/code-splitting.md [hunk 7]
```diff
@@ -329,8 +330,8 @@ T> Using webpackPreload incorrectly can actually hurt performance, so be careful
 Once you start splitting your code, it can be useful to analyze the output to check where modules have ended up. The [official analyze tool](https://github.com/webpack/analyse) is a good place to start. There are some other community-supported options out there as well:
 
 - [webpack-chart](https://alexkuz.github.io/webpack-chart/): Interactive pie chart for webpack stats.
-- [webpack-visualizer](https://chrisbateman.github.io/webpack-visualizer/): Visualize and analyze your bundles to see which modules are taking up space and which might be duplicates.
-- [webpack-bundle-analyzer](https://github.com/webpack-contrib/webpack-bundle-analyzer): A plugin and CLI utility that represents bundle content as convenient interactive zoomable treemap.
+- [webpack-visualizer](https://chrisbateman.github.io/webpack-visualizer/): Visualize and analyze your bundles to see which modules are taking up space and which are might be duplicates.
```
Comment (EugeneHlushko) [inline src/content/guides/code-splitting.md]:
```suggestion
- [webpack-visualizer](https://chrisbateman.github.io/webpack-visualizer/): Visualize and analyze your bundles to see which modules are taking up space and which might be duplicates.
```

Comment (EugeneHlushko) [reply src/content/guides/code-splitting.md]: I ment revert to original, it looks better to me than what we had, do you think we can remove that `are` ?

Comment (wizardofhogwarts) [reply src/content/guides/code-splitting.md]: @EugeneHlushko I think with `are` it's better. Though in my opinion. `are maybe` is better, but I am ready to revert if you say.

Comment (EugeneHlushko) [reply src/content/guides/code-splitting.md]: i've asked around, we can keep are

**Comment by EugeneHlushko** [comment]
Thanks!

**Pull request merged**